index.js

```diff
@@ -127,6 +127,41 @@ function reset () {
   moduleAliases = {}
 }
 
+//
+// Functions to load configurations from a file
+//
+
+// Load configuration from any file
+function loadConfigFile (packagePath) {
```

Comment:
Export it for unit testing purposes?

```suggestion
exports._loadConfigFile = function (configPath) {
```